packages/redux/src/entities/schemas/facet.ts

```diff
@@ -7,9 +7,9 @@ import { schema } from 'normalizr';
 
 export const getId = (
   { value, valueUpperBound, groupsOn }: FacetValue,
-  { key, type: parentType }: FacetGroup,
+  { key, type: parentType, hash }: FacetGroup,
 ) =>
-  `${key}_${value}${
+  `${key}_${value}_${hash}${
```

Comment:
The `hash` portion of the id must go at the end. If the facet is for a size or sizes by category for example, it will be inserted before the groupsOn portion.

Reply:
Right! Updated